cclient/iota_client_extended_api.c

```diff
@@ -235,3 +235,63 @@ retcode_t iota_client_get_account_data(iota_client_service_t const* const serv,
   get_balances_res_free(&blances_res);
   return ret_code;
 }
+
+retcode_t iota_client_find_transaction_objects(
+    iota_client_service_t const* const serv, hashes_t* addresses,
+    hashes_t const* const bundles, hashes_t const* const tags,
+    hashes_t const* const approvees, transaction_list_t* out_tx_objs) {
+  retcode_t ret_code = RC_OK;
+
+  log_debug(CCLIENT_EXTENDED_LOGGER_ID, "[%s:%d]\n", __func__, __LINE__);
+
+  find_transactions_req_t* find_tx_req = NULL;
+  find_transactions_res_t* find_tx_res = NULL;
+  get_trytes_req_t* get_trytes_req = get_trytes_req_new();
+  get_trytes_res_t* get_trytes_res = get_trytes_res_new();
+
+  if (!find_tx_req || !find_tx_res || !get_trytes_req || !get_trytes_res) {
+    ret_code = RC_CCLIENT_NULL_PTR;
+    goto done;
+  }
+
+  &find_tx_req->addresses = flex_hash_array_to_hash243_queue(
```

Comment:
syntax error!

Reply:
Could I just use hash243_queue_t as variable type of arguments of iota_client_find_transaction_objects() which includes addresses, bundles, etc?